vignettes/geologic-map.Rmd

```diff
@@ -39,46 +39,46 @@ install.packages("devtools")
 
 # Fetching
-First, we want to find the `strat_name_id`s that are associated with the Hell Creek, so we can grab all the available outcrop for the formation. We can do this by using some of the `def_` functions. These functions provide information about the information stored in the Macrostrat database. Let's have a look at the information associated with the Hell Creek Formation, using `def_strat_names()`:
+First, we want to find the `strat_name_id`s that are associated with the Hell Creek. These IDs are linked to unique stratigraphic names given to one or more geologic entities in the Macrostrat database. We can search for the relevant IDs by using some of the `def_*` functions. These functions provide information about the data stored in the database. Let's have a look at the information associated with the name Hell Creek, using `def_strat_names()`:
 
 ```{r def_strat_names, eval = TRUE}
-# Get information for the chosen formation
-hc_def <- def_strat_names(strat_name = "hell creek")
-hc_def
+# Search stratigraphic names for Hell Creek
+def_strat_names(strat_name = "hell creek")
 ```
-So we can see here that there are three different `strat_name`s associated with the Hell Creek. You'll also notice that these all contain the same `concept_id`, showing that this is one group of `strat_name`s which hopefully represent the same geologic entity. We can therefore use this `concept_id` to ensure we've got all mentions of the Hell Creek:
+We can see that there are three different `strat_name`s associated with the name Hell Creek. You'll also see that all three have the same `concept_id`, showing that these stratigraphic names are united under a single geologic entity. We can therefore use this `concept_id` to search for all references to the Hell Creek:
 
 ```{r check concept, eval = TRUE}
-# Get information for the chosen formation
+# Search stratigraphic names for all associated with Hell Creek concept_id
 def_strat_names(concept_id = 8598)
 ```
-Yup, that's all the entries for the Hell Creek. But hang on! Before going ahead and using these to pull our spatial data, you'll also notice that one of the `strat_name`s here is actually the Hell Creek Member, which belongs to the Lance formation. Let's try getting the information again, this time specifying the stratigraphic rank of "Formation":
+Yup, these three are the only entries for Hell Creek. But hang on! Before going ahead and using these to pull our spatial data, you'll also notice that one of the `strat_name`s here is actually the Hell Creek "Member", which is described as belonging to the Lance Formation. Let's try getting the information again, this time specifying the stratigraphic rank of "Formation":
 
 ```{r strat_name with rank, eval = T}
-# Get information for the chosen formation, specifying by stratigraphic rank
+# Search stratigraphic names for formations called Hell Creek
```

Comment:
It's not clear to me why we wouldn't want to include this 'Hell Creek Member' if it is considered part of the Hell Creek 'concept', can we add a sentence on why this is the right thing to do?

Reply:
Well, technically it's the Hell Creek member of the Lance formation; the Lance Formation is Wyoming's equivalent to the Hell Creek, but it's technically not the Hell Creek, and generally a member is going to be at a higher temporal resolution. I'll make sure I update the description. 

In response to your comment @willgearty, I think the concept intends to cove the Hell Creek Formation, but I think including the Hell Creek Member is a database error. I included this as I thought it was good to alert people to check their data for potential issues like this before they use it.

Reply:
Okay, done some more digging. Two things; 1) the Hell Creek Member is a term that is no longer in use, meaning that the Hell Creek Member is now officially part of the Hell Creek Formation, and 2) there is no difference in the resulting map removing it or adding it. So I'll just simplify this section and make reference to the fact that it's wise to check your list of formations for errors.